src/pages/Home/index.jsx

```diff
@@ -19,16 +19,16 @@ import styles from './index.scss';
 @observer
 export default class Home extends Component {
   static async onEnter({ categoryStore, appStore }, { category, search }) {
-    const params = {};
+    appStore.loadPageInit();
+    await categoryStore.fetchAll();
+
+    const params = { status: 'active' };
```

Comment:
define variable at the begining of function is better

Reply:
ok, params is used below method.

Reply:
yes, see javascript variable hoist